plugin-code/src/main/java/com/trivago/cluecumber/CluecumberReportPlugin.java

```diff
@@ -111,6 +119,11 @@ public void execute() throws CluecumberPluginException {
         logger.info("-----------------------------------------------");
         propertyManager.logProperties();
 
+        if (skip) {
```

Comment:
This should be before the property validation. Properties should not need to be checked if the plugin is skipped.